src/choiceitem.cpp

```diff
@@ -9,6 +9,10 @@ ChoiceItem::ChoiceItem(QString key, QString value, QWidget *parent) :
     m_value(value)
 {
     ui->setupUi(this);
+    QFile file(QString::fromUtf8(":/css/choiceBox.css"));
+    file.open(QFile::ReadOnly);
+    QString styleSheet = QString(file.readAll());
+    this->setStyleSheet(styleSheet);
```

Comment:
This four lines seems to be copied a lot of time, I wonder if it is time to move them in a helper function.